src/passport.js

```diff
@@ -43,19 +43,19 @@ export default class Passport {
 
       // If socket was already authenticated then re-authenticate
       // it with the server automatically.
-      if (socket.authenticated) {
-        const data = {
-          strategy: this.options.jwtStrategy,
-          accessToken: app.get('accessToken')
-        };
-        this.authenticateSocket(data, socket, emit)
-          .then(this.setJWT)
-          .catch(error => {
-            debug('Error re-authenticating after socket reconnect', error);
-            socket.authenticated = false;
-            app.emit('reauthentication-error', error);
-          });
-      }
+      // if (socket.authenticated) {
```

Comment:
If this isn't needed anymore, we should be able to remove the commented code.

Reply:
Ah... 0b6c016 should not be a part of this PR.  I should have used a different branch (forgot that github automatically adds commits).  It was pushed later in my fork as part of some testing to fix an error related to `NotAuthenticated: No auth token` when the client reconnects after the server has been restarted.  Commenting that line does indeed fix the problem i was having.

Let me know how you would like me to remedy.